### PR TITLE
Feat: Allow users to provide private key passphrase

### PR DESCRIPTION
**Pull request by srounce**
Users should be able to provide a passphrase in order to convert a private key which requires one.

### CODE REVIEW

File: cmd/ssh-to-age/main.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ func parseFlags(args []string) options {
 	f.BoolVar(&opts.privateKey, "private-key", false, "convert private key instead of public key")
 	f.StringVar(&opts.in, "i", "-", "Input path. Reads by default from standard output")
 	f.StringVar(&opts.out, "o", "-", "Output path. Prints by default to standard output")
+	f.StringVar(&opts.passphrase, "passphrase", "", "Output path. Prints by default to standard output")
```
Comment (Mic92) [inline cmd/ssh-to-age/main.go:25]:
```suggestion
	f.StringVar(&opts.passphrase, "passphrase", "", "Password used for decrypting the private ssh key")
```

File: cmd/ssh-to-age/main.go [hunk 2]
```diff
@@ -63,9 +64,12 @@ func convertKeys(args []string) error {
 		}
 		defer writer.Close()
 	}
-
 	if opts.privateKey {
```
Comment (Mic92) [inline cmd/ssh-to-age/main.go:67]: Thanks for the addition would it be possible to read the password using a function like this?

https://github.com/sapcc/nova-password/blob/6b3e3cf9951797da7d9dcf059245e063d23b6c41/main.go#L368

Than the password does not end up as likely in the the users shell history or a process that steals the secret from cmdline from `/proc`?

Comment (Mic92) [reply cmd/ssh-to-age/main.go:67]: I think `-password` would become a bool flag in this case.

Comment (srounce) [reply cmd/ssh-to-age/main.go:67]: Yeah can do, I've been using `read -s` for this so far but it's a reasonable change. Does it even need to be a bool flag in that case? Surely we could just check if the env var is set and if it is, use it as the password.

Comment (Mic92) [reply cmd/ssh-to-age/main.go:67]: That also works.

**Pull request merged**